app/views/sign_up/completions/show.html.erb

```diff
@@ -16,7 +16,7 @@
         ) %>
   </p>
   <div class='tablet:margin-left-1 tablet:margin-right-1' >
-    <%= render 'sign_up/completions/requested_attributes', requested_attributes: @view_model.requested_attributes_sorted %>
+    <%= render 'sign_up/completions/requested_attributes', requested_attributes: @presenter.pii %>
```

Comment:
in other places, requested_attributes is the names of the keys we want to render, not the actual PII, WDYT of renaming this to `pii`?

Reply:
I'll need to do some re-arranding w/i the presenter, but that could work